tagstudio/src/qt/ts_qt.py

```diff
@@ -65,20 +65,38 @@
 QSettings.setPath(QSettings.IniFormat, QSettings.UserScope, os.getcwd())
 
 
-def open_file(path: str):
+def open_file(path: str, file_manager: bool = False):
 	try:
 		if sys.platform == "win32":
-			# Windows needs special attention to handle spaces in the file
-			# first parameter is for title, NOT filepath
-			subprocess.Popen(["start", "", os.path.normpath(path)], shell=True, close_fds=True, creationflags=subprocess.DETACHED_PROCESS)
+			normpath = os.path.normpath(path)
+			if file_manager:
+				command_name = "explorer"
+				command_args = [f"/select,{normpath}"]
+			else:
+				command_name = "start"
+				# first parameter is for title, NOT filepath
+				command_args = ["", normpath]
+			subprocess.Popen([command_name] + command_args, shell=True, close_fds=True, creationflags=subprocess.DETACHED_PROCESS)
```

Comment:
`creationflags=subprocess.DETACHED_PROCESS` makes the file manager open behind TS.

Like this:
![image](https://github.com/TagStudioDev/TagStudio/assets/39923113/1d07f628-32ce-4bc0-b146-752f8ce0ed57)

Removing it makes the Windows Explorer open in front of TS, which, in my opinion, makes more sense.

Also, thanks for refactoring my code, didn't realize that there was an open_file function already. Great job!

Reply:
DETACHED_PROCESS is actually what makes the window keep open after TagStudio closes, which is especially something intended whenever opening in its respective program. These windows should not suddenly close when TagStudio is closed.
I'm not sure if there's any additional Windows flag for that, though.

Reply:
What happens when `subprocess.DETACHED_PROCESS | subprocess.CREATE_NEW_PROCESS_GROUP | subprocess.CREATE_BREAKAWAY_FROM_JOB` is used instead? That should further isolate these program launches.

Reply:
Resolved in 737be61.